make/package

```diff
@@ -25,7 +25,7 @@ mkdir -p ./output/packaging
 	cd ./output
 	(
 		cd ./packaging
-		curl -sS -L -O --fail http://d6r77u77i8pq3.cloudfront.net/releases/${traveling_ruby}
+		curl -sS -L -O --fail https://storage.googleapis.com/pelerinul/${traveling_ruby}
```

Comment:
Should last close to _forever_ :)
Don't have anything else ATM.
I tried it and it didn't work - because of the rubygems version.
Traveling ruby is dead anyway - we'll need something else. See https://trello.com/c/vfg6gKVO